src/test/resources/config/application.yml

```diff
@@ -100,6 +102,14 @@ jhipster:
   mail:
     from: test@localhost
     base-url: http://127.0.0.1:8080
+  security:
+    authentication:
+      jwt:
+        # This token must be encoded using Base64 and be at least 256 bits long (you can type `openssl rand -base64 64` on your command line to generate a 512 bits one)
+        base64-secret: NGUwNTlhNDNhOWQ2ZGJlMmEzODczOWJjYWM2MWY4NDAwOTM4YTBmNGFjM2UxNjczZDU2YWZjZjc3MjdkNmU4YjVhZmIwYzI1NDhjMDFkZThiYmE5Mjc5MjM0MGVhODU5MGFhMTAyOTE0M2I3ODA1MGUzZWFhYWUwNWY2ZDgyNjQ=
```

Comment:
is it necessary to add to application.yml? Can the test use it's own secret?

Reply:
Since this is the application.yml used for all tests, I think it would make sense for the secret to live here. This secret is also already stored in application-prod.yml and application-dev.yml, so I figured the information wasn't sensitive.

However, I could look into disabling authentication for tests if that is what you think is best.